.gitignore

```diff
@@ -46,4 +46,6 @@ npm-debug.log*
 .tern-project
 x-pack/plugins/apm/tsconfig.json
 apm.tsconfig.json
+/x-pack/legacy/plugins/apm/e2e/snapshots.js
```

Comment:
is this still needed now that the migration is done

Reply:
Could just have been a bad merge.  Can I remove it?

Reply:
i think it was valid, but yeah if you still have changes for that file they can be git cleaned now instead.

Reply:
Oh I personally have nothing to do with that file.